### PR TITLE
Modified my changes to package.json

### PR DESCRIPTION
**Pull request by Taguiar**
I did some changes and I hope you like it.
There is something wrong during compilation, but it is not with my modification.

Best regards,

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
     "name": "bugsnag-js",
+	"description": "The Bugsnag is an error reporter for JavaScript that automatically detects and reports errors.",
```
Comment (wordofchristian) [inline package.json]: I would remove the word "The" when referring to Bugsnag. I think this should read.

> Automatically detect JavaScript errors on your websites, collect detailed diagnostic information, and get notified about errors in real-time.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+	"keywords":[
+		"exception",
+		"bugsnag",
+		"error",
+		"JavaScript"
```
Comment (wordofchristian) [inline package.json]: I would add "browsers" and "monitoring" here

File: package.json (hunk 1 above)
Comment (wordofchristian) [inline package.json]: Also please convert tabs to spaces. We use 2 space indents.

**Comment by kattrali** [comment]
Updated the PR with some suggested changes and merged manually. Thanks again, @Taguiar!

**Pull request closed without merging**